pkg/scalers/github_runner_scaler.go

```diff
@@ -41,19 +39,19 @@ type githubRunnerScaler struct {
 }
 
 type githubRunnerMetadata struct {
-	githubAPIURL              string
-	owner                     string
-	runnerScope               string
-	personalAccessToken       *string
-	repos                     []string
-	labels                    []string
-	noDefaultLabels           bool
-	enableEtags               bool
-	targetWorkflowQueueLength int64
-	triggerIndex              int
-	applicationID             *int64
-	installationID            *int64
-	applicationKey            *string
+	GithubAPIURL              string   `keda:"name=githubApiURL, order=triggerMetadata;resolvedEnv, default=https://api.github.com"`
+	Owner                     string   `keda:"name=owner, order=triggerMetadata;resolvedEnv"`
+	RunnerScope               string   `keda:"name=runnerScope, order=triggerMetadata;resolvedEnv"`
```

Comment:
What do you think, is the use of [enum](https://github.com/kedacore/keda/blob/main/pkg/scalers/scalersconfig/typed_config.go#L113-L114) possible to use for “org”, “ent” or “repo”?

```suggestion
	RunnerScope               string   `keda:"name=runnerScope, order=triggerMetadata;resolvedEnv, enum=org;ent;repo"`
```

Reply:
Pushed a fix for runnerscope enums